calour/plotting.py

```diff
@@ -422,3 +427,51 @@ def plot_stacked_bar(exp: Experiment, field=None, sample_color_bars=None, color_
     fig.tight_layout()
     fig.subplots_adjust(hspace=0.01, wspace=0.01)
     return fig
+
+
+def plot_scatter_matrix(exp: Experiment, field, feature_ids, title_field=None,
+                       transform_y=None, transform_x=None,
+                       ncols=5, nrows=6, figsize=(10, 12)):
```

Comment:
do we want ncols,nrows hard set to 5,6? or maybe can do automatic numbers by default (i.e. when None), based on number of features (i.e. sqrt or similar)?
also, figsize by default maybe 1? or is it too small?